packages/common/src/utils/projectMemberRole.ts

```diff
@@ -110,3 +110,25 @@ export const convertProjectRoleToSpaceRole = (
             );
     }
 };
+
+export const convertProjectRoleToOrganizationRole = (
+    projectRole: ProjectMemberRole,
+): OrganizationMemberRole => {
+    switch (projectRole) {
+        case ProjectMemberRole.VIEWER:
+            return OrganizationMemberRole.VIEWER;
+        case ProjectMemberRole.INTERACTIVE_VIEWER:
+            return OrganizationMemberRole.VIEWER;
+        case ProjectMemberRole.EDITOR:
+            return OrganizationMemberRole.EDITOR;
+        case ProjectMemberRole.DEVELOPER:
+            return OrganizationMemberRole.EDITOR;
```

Comment:
there is a developer role on an organization level

Reply:
uhh, thanks

Reply:
I copied this from `convertProjectRoleToSpaceRole`  and I didn't even look into the `cases` :see_no_evil: